### PR TITLE
AD review:  clarify extensibility of "intuse" claim

### PR DESCRIPTION
**Pull request by gmandyam**
See https://github.com/ietf-rats-wg/eat/issues/379

### CODE REVIEW

File: draft-ietf-rats-eat.md [hunk 1]
```diff
@@ -1174,6 +1174,8 @@ See {{common-types}} for OID and URI encoding.
 EAT's may be used in the context of several different applications.  The "intuse"
 claim provides an indication to an EAT consumer about  the intended usage
 of the token. This claim can be used as a way for an application using EAT to internally distinguish between different ways it uses EAT.
+5 possible values for "intuse" are currently defined indicated 5 different intended use indications, but additional values above 5 can
+be defined and registered pending IANA-administered expert review (see {{iana-cons}}).
```
Comment (carl-wallace) [inline draft-ietf-rats-eat.md:1178]: EAT's should be EATs in the first sentence. I'd change the wording on the new language slightly:

```
Five possible values for "intuse" are currently defined, providing 5 different intended use indications. However, additional values can be defined and registered pending IANA-administered expert review (see {{iana-cons}}).
```

I don't think just referencing "expert review" answers the question regarding a registry, so I'd also add something about a possible future registry.

```
At present, there is not a registry for "intuse" values. If a future Internet Draft defines any additional values, a registry should be created and initialized with the five values defined in this document plus any new values.
```

Comment (gmandyam) [reply draft-ietf-rats-eat.md]: Please review updates

File: draft-ietf-rats-eat.md [hunk 2]
```diff
@@ -1171,11 +1171,12 @@ See {{common-types}} for OID and URI encoding.
 
 ### intuse (Intended Use) Claim
 
-EAT's may be used in the context of several different applications.  The "intuse"
+EATs may be used in the context of several different applications.  The "intuse"
 claim provides an indication to an EAT consumer about  the intended usage
 of the token. This claim can be used as a way for an application using EAT to internally distinguish between different ways it uses EAT.
 5 possible values for "intuse" are currently defined indicated 5 different intended use indications, but additional values above 5 can
```
Comment (quic-jodonogh) [inline draft-ietf-rats-eat.md:1177]: "additional values above 5" reads slightly awkwardly to me. Maybe just "additional values"?

File: draft-ietf-rats-eat.md (hunk 1 above)
Comment (carl-wallace) [reply draft-ietf-rats-eat.md]: Works for me. Thanks.

File: draft-ietf-rats-eat.md (hunk 2 above)
Comment (gmandyam) [reply draft-ietf-rats-eat.md]: @quic-jodonogh change made.

**Comment by laurencelundblade** [comment]
Agree that it should be possible to define new values.

I don't know if we can get away with saying that a registry can be created in the future, but am fine trying that.

Another thing possible here might be to say values above 64K are proprietary and below are reserved for a future IANA registry.

Agree with the wording suggestions.

**Comment by carl-wallace** [comment]
There's a draft in LAMPS that has language on establishing a registry in the future (draft-ietf-lamps-header-protection).

File: draft-ietf-rats-eat.md [hunk 3]
```diff
@@ -1171,9 +1171,12 @@ See {{common-types}} for OID and URI encoding.
 
 ### intuse (Intended Use) Claim
 
-EAT's may be used in the context of several different applications.  The "intuse"
+EATs may be used in the context of several different applications.  The "intuse"
 claim provides an indication to an EAT consumer about  the intended usage
 of the token. This claim can be used as a way for an application using EAT to internally distinguish between different ways it uses EAT.
```
Comment (laurencelundblade) [inline draft-ietf-rats-eat.md:1176]: Wording could improve here with less use of "use".

File: draft-ietf-rats-eat.md [hunk 4]
```diff
@@ -1171,9 +1171,12 @@ See {{common-types}} for OID and URI encoding.
...
+5 possible values for "intuse" are currently defined indicated 5 different intended use indications, but additional values can
+be defined and registered pending IANA-administered expert review (see {{iana-cons}}).  This could involve the future creation of a registry
+for additional values.
```
Comment (laurencelundblade) [inline draft-ietf-rats-eat.md:1179]: Could improve the wording here and shorten a lot I think. Something like An IANA registry could be created to extend past the 5 values defined here."

**Review by laurencelundblade — CHANGES_REQUESTED** [review]
Agree in concept, but please make a few wording improvements.

File: draft-ietf-rats-eat.md (hunk 4 above)
Comment (gmandyam) [reply draft-ietf-rats-eat.md]: Please see latest changes

**Review by laurencelundblade — APPROVED** [review]

**Pull request merged**